app/templates/notify-settings--reply-to.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'page-with-header.html' %}
+{% block title %}Choose options{% endblock %}
+{% block head %}{{ super() }}{% endblock %}
+
+{% block breadcrumbs %}
+ <li class="breadcrumbs--list-item">
+   <a class="breadcrumbs--link" href="/">Mentor matcher</a>
+ </li>
+{% endblock %}
+
+{% block pagecaption %}Mentor matcher{% endblock %}
+{% block pageheading %}Notify participants{% endblock %}
+{% block pageexcerpt %}Send emails to mentors and mentees with the results of the matching process.{% endblock %}
+
+{%  block content %}
+<div class="grid-row">
+  <div class="grid-column-two-thirds" id="options">
+  
+    <form method=post enctype=multipart/form-data action="">
+      {% if error_message %}
+      <blockquote class="error warning-text">
+        <p><strong>An error has occurred</strong></p>
+        <p>{{ error_message }}</p>
+      </blockquote>
+      {% endif %}
+      <div class="form-group">
+      
+        <fieldset id="fieldset-notify" aria-describedby="fieldset-notify--hint" class="fieldset" style="margin-bottom: 45px">
+          <legend id="fieldset-notify--title" class="legend-md">
+          Reply-to email address
+          </legend>
+          
+          <!-- Reply ID isn't necessary if the Notify account only has 1 reply-to email address set up -->
```

Comment:
I'm not sure about this content - I think it implies that you need to have a reply-to id. Can we use a conditional pop-out box, like the [conditional radio](https://design-system.service.gov.uk/components/radios/) ?

Reply:
The issue with using a so called conditional radio is that it's not a native feature, so it would require bespoke javascript to implement. I've tested it with javascript disabled on the GOV.UK design system elements, and if js is disabled then the field just doesn't show up. 

Given our previous reluctance to add js to manipulate the interface itself, this would seem to be taking an inconsistent approach!

My suggestion would be that either:

(a) we don't include this step – in this scenario, Notify should fallback on whatever the default is in the Notify system for that individual account; it's described as optional in the API docs
(b) we only show this page if we know there is more than one reply-to email set up – but I don't know if that is possible?

What do you think

Reply:
Ach I didn't realise that. As far as I can tell from the docs, the client doesn't let us see what reply to options a specific key has.

Maybe we just change the content a little?

## Alternative reply-to address

If you'd like users to reply to a specific email address, you'll need to add it to Notify. Once you've done that, put the corresponding key here. This step is optional. If you don't put anything in here, the reply-to address will be whatever your default is in Notify

what do you think?

Reply:
Have amended in latest commit...